packages/core/parcel-bundler/test/integration/babel-plugin-autoinstall/package.json

```diff
@@ -1,3 +1,6 @@
 {
-  "name": "babel-plugin-autoinstall"
+  "name": "babel-plugin-autoinstall",
+  "devDependencies": {
+    "@babel/core": "^7.1.2"
+  }
```

Comment:
this change shouldn't be committed, it's part of the test to verify that it was added.

Reply:
Ow damn, I'll remove it and add it to a gitignore to prevent this in the future